cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -69,23 +70,29 @@ message Options {
 	// host and UVM.
 	bool scale_cpu_limits_to_sandbox = 11;
 
-	// default_container_scratch_size_in_gb is the default scratch size (sandbox.vhdx) 
+	// default_container_scratch_size_in_gb is the default scratch size (sandbox.vhdx)
 	// to be used for containers. Every container will get a sandbox of `size_in_gb` assigned
-	// instead of the default of 20GB. 
+	// instead of the default of 20GB.
 	int32 default_container_scratch_size_in_gb = 12;
 
-	// default_vm_scratch_size_in_gb is the default scratch size (sandbox.vhdx) 
-	// to be used for the UVM. This only applies to WCOW as LCOW doesn't mount a scratch 
+	// default_vm_scratch_size_in_gb is the default scratch size (sandbox.vhdx)
+	// to be used for the UVM. This only applies to WCOW as LCOW doesn't mount a scratch
 	// specifically for the UVM.
 	int32 default_vm_scratch_size_in_gb = 13;
 
-	// share_scratch specifies if we'd like to reuse scratch space between multiple containers. 
+	// share_scratch specifies if we'd like to reuse scratch space between multiple containers.
 	// This currently only affects LCOW. The sandbox containers scratch space is re-used for all
-	// subsequent containers launched in the pod.  
+	// subsequent containers launched in the pod.
 	bool share_scratch = 14;
-	//NCProxyAddr is the address of the network configuration proxy service. If omitted 
-	// the network is setup locally. 
-	string NCProxyAddr = 15; 
+
+	// NCProxyAddr is the address of the network configuration proxy service. If omitted
+	// the network is setup locally.
+	string NCProxyAddr = 15;
+
+	// log_level specifies the logrus log level for the shim. Supported values are a string representation of the
+	// logrus log levels: "trace", "debug", "info", "warn", "error", "fatal", "panic". This setting will override
```

Comment:
do we want to make an enum for this?

Reply:
I didn't want to have to supply 0-n for the setting in the containerd config, it seems more intuitive to supply "debug", "warning" etc. Also makes it so you can feed it directly to logrus

Reply:
yeah that's fine